app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,7 @@
 
   <body>
     <% if current_user %>
+      <%= link_to "Members", users_path %>
       <%= link_to "Sign out", session_path("current"), method: "delete" %><br>
       Logged in as <%= current_user.email %>.<br>
```

Comment:
Should we change the current_user.email to current_user.username for the "Logged in as" message?

Reply:
I have changed this on another branch :)